service-model-generator/modules/service-model-generator-ls-extension/src/main/java/io/ballerina/servicemodelgenerator/extension/util/TypeCompletionGenerator.java

```diff
@@ -166,7 +181,33 @@ public class TypeCompletionGenerator {
                 "5XX", "Not Extended", "http:NotExtended", "510"));
         defaultResponses.add(new TypeCompletion(
                 "5XX", "Network Authentication Required", "http:NetworkAuthenticationRequired", "511"));
+
+        // GraphQL Scalar types
+        inputTypes.add(new TypeCompletion(GRAPHQL_SCALAR_TYPE, "int", "int", ""));
```

Comment:
```suggestion
        inputTypes.add(new TypeCompletion(GRAPHQL_SCALAR_TYPE, "int", "int", null));
```
We can pass null here since we don't need the status code.